core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aas/AspectModelAasVisitor.java

```diff
@@ -555,8 +564,9 @@ private <T extends Collection> Environment visitCollectionProperty( final T coll
                .displayName( LangStringMapper.NAME.map( property.getPreferredNames() ) )
                .description( LangStringMapper.TEXT.map( property.getDescriptions() ) )
                .value( List.of( decideOnMapping( property, context ) ) )
-               .typeValueListElement( AasSubmodelElements.SUBMODEL_ELEMENT )
-               .supplementalSemanticIds( buildGlobalReferenceForSeeReferences( collection ) );
+               .typeValueListElement( AasSubmodelElements.SUBMODEL_ELEMENT_COLLECTION )
```

Comment:
I think here the same comment applies that @andreas-schilling previously mentioned; the typeValue for the elements in the list should not be submodel element collection.

Reply:
My original finding was: it should not be submodel_element but the kind that is contained in the list, might be a submodel collection, might be a property, it depends